lib/rabbitmq/factory.js

```diff
@@ -22,14 +23,20 @@ const Factory = (config) => {
 
   return {
 
-    send: (name, msg, options = {}) =>
-      getQueue(name).send(options, msg)
+    send: (name, msg) =>
+      Bluebird.resolve(getQueue(name))
```

Comment:
getQueue(name) was returning a promise so there wasn't a send property of it. I used Bluebird so I can have .get